lightkube/core/client.py

```diff
@@ -416,7 +417,7 @@ def log(self, name, *, namespace=None, container=None, follow=False,
         req = self._client.build_adapter_request(br)
         resp = self._client.send(req, stream=follow)
         self._client.raise_for_status(resp)
-        return resp.iter_lines()
+        return map(transform_newlines(newlines), resp.iter_lines())
```

Comment:
Same consideration as above:

```suggestion
        return (line + '\n' if newlines else line for l in resp.iter_lines())
```